src/library/scala/collection/mutable/ArrayOps.scala

```diff
@@ -202,7 +202,7 @@ object ArrayOps {
   }
 
   /** A class of `ArrayOps` for arrays containing `byte`s. */
-final class ofByte(override val repr: Array[Byte]) extends AnyVal with ArrayOps[Byte] with ArrayLike[Byte, Array[Byte]] {
+  final class ofByte(override val repr: Array[Byte]) extends AnyVal with ArrayOps[Byte] with ArrayLike[Byte, Array[Byte]] {
```

Comment:
The class comments should read "A subclass...", and the element type should use the correct name such as "containing `Byte`s." Also there are extraneous empty lines around line 180.